configure

```diff
@@ -2962,15 +2962,16 @@ if test "$linux" = "yes" ; then
 fi
 
 ##########################################
-# Epoxy
+# epoxy probe
 
 cat > $TMPC <<EOF
 #include <epoxy/gl.h>
 int main(void) { return 0; }
 EOF
 epoxy_libs="-lepoxy"
 if ! compile_prog "" "$epoxy_libs" ; then
-    error_exit "epoxy check failed. Please install epoxy/libepoxy with your package manager and try again."
+    error_exit "epoxy check failed" \
+        "Make sure to have the epoxy libs and headers installed."
```

Comment:
This is better but I think can be more succinct. Here's an example from the pixman check which we should follow:

```
  error_exit "pixman >= 0.21.8 not present." \
      "Please install the pixman devel package."
```

So basically replace `pixman` with epoxy, and you can probably drop the `>= 0.21.8` part.